jina/resources/executors._merge_matches_topk.yml

```diff
@@ -0,0 +1,17 @@
+!BaseExecutor
+with: {}
+metas:
+  name: merge_matches_topk
+requests:
+  on:
+    [SearchRequest, TrainRequest, IndexRequest, DeleteRequest, UpdateRequest]:
+      - !ReduceAllDriver
+        with:
+          traversal_paths: ['m']
+      - !SliceQL
+        with:
+          start: 0
+          end: $JINA_TOPK
```

Comment:
this has to be set arbitrarily and overriden by QueryLangDriver, but setting an env variable is bad since we do not know it will exist. you can write 10

Reply:
the new syntax for env var substituion  is https://docs.jina.ai/chapters/yaml/substitute.html#confval-${{ENV.var}}

Reply:
but we should not consider this is there right?